sphinx/writers/latex.py

```diff
@@ -1853,28 +1853,14 @@ def visit_figure(self, node):
                              (node['align'] == 'right' and 'r' or 'l', length or '0pt'))
             self.context.append(ids + '\\end{wrapfigure}\n')
         elif self.in_minipage:
-            if ('align' not in node.attributes or
-                    node.attributes['align'] == 'center'):
-                self.body.append('\n\\begin{center}')
-                self.context.append('\\end{center}\n')
-            else:
-                self.body.append('\n\\begin{flush%s}' % node.attributes['align'])
-                self.context.append('\\end{flush%s}\n' % node.attributes['align'])
+            self.body.append('\n\\begin{center}')
+            self.context.append('\\end{center}\n')
```

Comment:
the align attribute has already been tested at https://github.com/sphinx-doc/sphinx/pull/4429/files#diff-81c44cde8d80608ae558b93878f9be8aR1846